paymentsheet/src/main/java/com/stripe/android/link/confirmation/DefaultLinkConfirmationHandler.kt

```diff
@@ -0,0 +1,107 @@
+package com.stripe.android.link.confirmation
+
+import com.stripe.android.core.Logger
+import com.stripe.android.core.strings.resolvableString
+import com.stripe.android.link.LinkConfiguration
+import com.stripe.android.link.model.LinkAccount
+import com.stripe.android.model.ConsumerPaymentDetails
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.paymentelement.confirmation.ConfirmationHandler
+import com.stripe.android.paymentelement.confirmation.PaymentMethodConfirmationOption
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.state.PaymentElementLoader
+import javax.inject.Inject
+
+internal class DefaultLinkConfirmationHandler @Inject constructor(
+    private val configuration: LinkConfiguration,
+    private val logger: Logger,
+    private val confirmationHandler: ConfirmationHandler
+) : LinkConfirmationHandler {
+    override suspend fun confirm(
+        paymentDetails: ConsumerPaymentDetails.PaymentDetails,
+        linkAccount: LinkAccount
+    ): Result {
+        return kotlin.runCatching {
```

Comment:
```suggestion
        return runCatching {
```

Is it possible to write this this way? Or why do we need to specify `kotlin` here?